doc/js-api/remotestorage.rst

```diff
@@ -76,6 +76,14 @@ List of events
 """""""""""""
    Emitted when a wire request completes
 
+``sync-req-done``
+"""""""""""""""""
+   Emitted when a single sync task has finished
+
+``sync-done``
+"""""""""""""
+   Emitted when the complete sync cycle has finished
```

Comment:
What's "the" here? As this document describes the events emitted by rs in general, what is "the complete sync cycle"? I think this is supposed to mean a/one sync cycle/process?